### ISSUE TITLE
USB installer stuck on boot (Sonoma)

### ISSUE THREAD

**Issue by arjepsen** [chris1111/HP-Probook-EliteBook-Package-Creator-OC#203]
Hey.
I am trying to use your tool to install Sonoma on a 8470p.
I first did a opencore install of Big Sur, and used that system to create the installation usb, per your instructions.

Then I reboot the pc.
I cannot choose the installation usb drive for booting, using F9 during boot, but I am able to use the old opencore boot section to choose the installation usb for boot.
The Apple logo screen shows, and the progress bar runs up to the middle, but stops there, and dont go any further.

Do you have any idea about what I might be doing wrong?
Regards.
Anders

**Comment by chris1111** [comment]
If you cannot use the usb key it is because you did not do things correctly or your usb key is not good or sometimes another usb port can solve the selection of the usb key

**Comment by arjepsen** [comment]
I followed this video:
https://www.youtube.com/watch?v=gaxkglymYkE
Just using Sonoma instead, and choosing 7 series laptop instead of 8.
Only difference I can see, is that after running the "Install Media HP Laptop OC" utility, in the video it seems the usb has a seperate EFI partition. After I ran the utility, there is only one partition, but it contains the EFI folder, along with the "Install macOS Snomoa" and the Private folder.
I then just copy the HP-ProBook-EliteBook-macOS.pkg file, the OpenCore-Patcher, and the GateKeeper to the root of the usb drive, umount, and reboot.
When I reboot, I can't choose the usb drive itself in the bios boot menu (F9), but since opencore is installed for the BigSur installation that I used for creating the usb drive, I can boot the usb installer from that boot-menu. 
Then as described, the apple logo is shown, and the progress bar goes up to about half way, and then stops there.
(left it for 30 minutes, but no more progress.)

So it seems the biggest difference to the video, is that the EFI folder is not on a seperate EFI partition on the usb installer?
I wonder if this is pointing to something that has gone wrong?
